lib/elixir/lib/exception.ex

```diff
@@ -740,7 +740,7 @@ defmodule Exception do
       ""
 
   """
-  @spec format_file_line(term, term, String.t()) :: String.t()
+  @spec format_file_line(String.t() | nil, non_neg_integer | nil, String.t()) :: String.t()
```

Comment:
I don't think `line` or `column` can be `0`, so I think we want

```suggestion
  @spec format_file_line(String.t() | nil, pos_integer | nil, String.t()) :: String.t()
```

here and below.

Reply:
they can be zero, line 0 usually means we don't have a precise line too (it is used by erlang in places where a line must be given).